frontend/src/container/SearchResult.jsx

```diff
@@ -19,15 +19,15 @@ import ContentItemHeader from '../component/Workspace/ContentItemHeader.jsx'
 import {
```

Comment:
Shouldn't this file be renamed `SimpleSearch` or something?

Reply:
will fix